tests/config.json

```diff
@@ -4,17 +4,25 @@
       "model_name": "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
       "model_class": "LlamaForCausalLM"
     },
-    {
-      "model_name": "gpt2",
-      "model_class": "GPT2LMHeadModel"
-    },
     {
       "model_name": "Salesforce/codegen-350M-mono",
       "model_class": "CodeGenForCausalLM"
     },
     {
-      "model_name": "wtang06/mpt-125m-c4",
-      "model_class": "MPTForCausalLM"
+      "model_name": "microsoft/Phi-3-mini-4k-instruct",
+      "model_class": "Phi3ForCausalLM"
+    },
+    {
+      "model_name": "tiiuae/falcon-7b",
+      "model_class": "FalconForCausalLM"
+    },
+    {
+      "model_name": "Qwen/Qwen2-0.5B",
+      "model_class": "Qwen2ForCausalLM"
+    },
+    {
+      "model_name": "bigcode/starcoder2-3b",
```

Comment:
can we add one for starcoder 15,5B?

Reply:
The idea was to test the pipeline with smaller model checkpoints to get the output and execution correctness. 
I can add one larger set for starcoder2. Maybe. [bigcode/starcoder2-7b](https://huggingface.co/bigcode/starcoder2-7b), is that fine?